src/cds_objects.cc

```diff
@@ -126,7 +126,7 @@ std::shared_ptr<CdsObject> CdsObject::createObject(unsigned int objectType)
     } else if (IS_CDS_ITEM(objectType)) {
         obj = std::make_shared<CdsItem>();
     } else {
-        throw_std_runtime_error("invalid object type: " + fmt::to_string(objectType));
+        throw_std_runtime_error(fmt::format("invalid object type: {}", objectType));
```

Comment:
I recommend updating `throw_std_runtime_error(msg)` in `exceptions.h` to include `fmt::format`. Make code a lot clearer and remove another bunch of string concats!

Reply:
That's already the case with this PR.

Reply:
I was thinking about getting rid of the nesting.

Reply:
Maybe you can add a second macro  and we reduce it to nested macros.

```
#define throw_runtime_error_with_message(...) throw_std_runtime_error(fmt::format(__VA_ARGS__))
```

Reply:
fmt happily handles the "no-args" case, so just updating the existing macro should work